lib/processes.js

```diff
@@ -61,6 +61,21 @@ function isChromeInDocker(browserName) {
   });
 }
 
+function mergeChromeOptions(baseOpts, configOpts) {
+  var reg = /^([-\w]+)=?([\.,\/\w]*)/;
+
+  var objOpts = baseOpts.concat(configOpts).reduce(function filterOpts(acc, val) {
+    var match = reg.exec(val);
+    acc[match[1]] = match[2];
+    return acc;
+  }, {});
+
+  return Object.entries(objOpts).reduce(function mergeOpts(acc, opt) {
```

Comment:
`Object.entries()` requires node 8+; at the moment we want to maintain compatibility with 6+; you can use `Object.keys()` and a bit more work and maintain compatibility